Add osu! Philippines Nationals 2022-2023 wiki article

topecnz opened the pull request:
~Note: oPHN 2021 wiki articles still need to be added soon.~
Addition: I've edited some information in 2019_4K

Edit: Some old beatmap urls are already deleted and not recoverable (2019_4K)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

Walavouchey commented:
i've converted this to a draft because of all the placeholders. mark as ready when ready

topecnz commented:
Skip oPHN 2021 for now. Gonna add more info on next updates for PHNC23.

Walavouchey reviewed (changes requested):
- the 2021 articles are nowhere near complete (see the [tournament template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md)). i wouldn't merge them as they are
- "osu!standard" is unofficial terminology. if it's part of the tournament name, it can stay as is, but anywhere else, the game mode should be called "osu!"
- why are the rulesets so different between tournaments? 🤔 
- your current structure is

```markdown
- wiki/Tournaments/oPHN
  - 2019_4K
  - 2021
      - osu
      - mania
  - 2022
      - taiko
      - catch
      - mania
  - 2023
      - osu
      - taiko
      - catch
      - mania
```

but it should be

```markdown
- wiki/Tournaments/oPHN
  - 2019_4K
  - 2021_mania
  - 2021_osu
  - 2022_taiko
  - 2022_catch
  - 2022_mania
  - 2023_osu
  - 2023_taiko
  - 2023_catch
  - 2023_mania
```

to align with other tournaments

- as i mentioned on discord, please split this pr into one for the ongoing tournaments (2023) and one for the past tournaments. this will otherwise take a while to review

topecnz commented:
> * the 2021 articles are nowhere near complete (see the [tournament template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md)). i wouldn't merge them as they are

Just removed oPHN 2021

> * "osu!standard" is unofficial terminology. if it's part of the tournament name, it can stay as is, but anywhere else, the game mode should be called "osu!"

> * why are the rulesets so different between tournaments? 🤔

afaik, it depends to the game mode hosts and players' suggestions/feedback

> * your current structure is
> 
> ```
> - wiki/Tournaments/oPHN
>   - 2019_4K
>   - 2021
>       - osu
>       - mania
>   - 2022
>       - taiko
>       - catch
>       - mania
>   - 2023
>       - osu
>       - taiko
>       - catch
>       - mania
> ```
> 
> but it should be
> 
> ```
> - wiki/Tournaments/oPHN
>   - 2019_4K
>   - 2021_mania
>   - 2021_osu
>   - 2022_taiko
>   - 2022_catch
>   - 2022_mania
>   - 2023_osu
>   - 2023_taiko
>   - 2023_catch
>   - 2023_mania
> ```
> 
> to align with other tournaments

Just changed

> * as i mentioned on discord, please split this pr into one for the ongoing tournaments (2023) and one for the past tournaments. this will otherwise take a while to review

Gonna do it rn.

topecnz reviewed (commented):
comments in: https://github.com/ppy/osu-wiki/pull/10331#issuecomment-1793425342

Walavouchey commented on wiki/Tournaments/oPHN/2022_mania/en.md:
```diff
@@ -0,0 +1,787 @@
+- [Spreadsheets](https://docs.google.com/spreadsheets/d/1_wM5fumI4LpKtIxio0evxpkts3XLCuaQrhX-YWQKbUI/)
+- [Twitch Livestream](https://www.twitch.tv/osuliveph)
+
+## Participants (osu!mania 4K)
```
```suggestion
## Participants

### osu!mania 4K
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_mania/en.md:
```diff
@@ -0,0 +1,787 @@
+- ::{ flag=PH }:: [KnfeConcorde](https://osu.ppy.sh/users/18390445)
+- ::{ flag=PH }:: [SZ20](https://osu.ppy.sh/users/28352929)
+
+## Participants (osu!mania 7K)
```
```suggestion
### osu!mania 7K
```

Walavouchey commented on wiki/Tournaments/oPHN/2019_4K/en.md:
```diff
@@ -62,122 +66,172 @@ The **osu! Philippines Nationals 2019 (osu!mania 4K)** (***PHN4K 2019***) is a o
 
 ## Mappools
 
+### Finals Week 2
```
```suggestion
### Finals week 2
```

Walavouchey commented on wiki/Tournaments/oPHN/2019_4K/en.md:
```diff
@@ -62,122 +66,172 @@ The **osu! Philippines Nationals 2019 (osu!mania 4K)** (***PHN4K 2019***) is a o
 
 ## Match results
 
+### FInals Week 2
```
```suggestion
### FInals week 2
```

Walavouchey commented on wiki/Tournaments/oPHN/2019_4K/en.md:
```diff
@@ -62,122 +66,172 @@ The **osu! Philippines Nationals 2019 (osu!mania 4K)** (***PHN4K 2019***) is a o
+| Silhoueska Elze | 4 | **7** | **- Kura -** | [#1](https://osu.ppy.sh/community/matches/53330098) |
+| **- Kura -** | **7** | 4 | Silhoueska Elze | [#1](https://osu.ppy.sh/community/matches/53332229) |
+
+### Finals Week 1
```
```suggestion
### Finals week 1
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+
+## Mappool
+
+### Grand finals
```
```suggestion
### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+- Tiebreaker 
+  1. [Silentroom - Nhelv (Bunnrei) [Aftermath]](https://osu.ppy.sh/beatmapsets/1755429#fruits/3592362)
+
+## Finals
```
```suggestion
### Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+
+## Match results
+
+### Grand finals
```
```suggestion
### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+- ::{ flag=AU }:: [shorts](https://osu.ppy.sh/users/8021372)
+- ::{ flag=PH }:: [shadowkun-](https://osu.ppy.sh/users/13450837)
+
+## Poduim
```
```suggestion
## Podium
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+
+### 4K
+
+#### Grand finals
```
```suggestion
#### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+
+### 7K
+
+#### Grand finals
```
```suggestion
#### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+
+### 4K
+
+#### Grand finals
```
```suggestion
#### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+| Vulpsi ::{ flag=PH }:: | 0 | **7** | ::{ flag=PH }:: **[KN]Puddles** | *win by default* |
+| Dyei ::{ flag=PH }:: | 0 | **7** | ::{ flag=PH }:: **bojii the rock** | [#1](https://osu.ppy.sh/community/matches/109278242) |
+
+#### Semiinals
```
```suggestion
#### Semifinals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+
+### 7K
+
+#### Grand finals
```
```suggestion
#### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+## Mappools
+
+### Grand finals
```
```suggestion
### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+## Match results
+
+### Grand finals
```
```suggestion
### Grand Finals
```

Walavouchey commented on wiki/Tournaments/oPHN/en.md:
```diff
@@ -3,4 +3,10 @@
 Index page for all of the osu! Philippines Nationals series.
 
 - [osu! Philippines Nationals 2019 - osu!mania 4K](2019_4K)
+- [osu! Philippines Nationals 2022 - osu!catch](2022_catch)
+- [osu! Philippines Nationals 2022 - osu!mania](2022_mania)
+- [osu! Philippines Nationals 2022 - osu!taiko](2022_taiko)
+- [osu! Philippines Nationals 2023 - osu!](2023_osu)
+- [osu! Philippines Nationals 2023 - osu!mania](2023_mania)
+- [osu! Philippines Nationals 2023 - osu!taiko](2023_taiko)
 - [osu! Philippines Nationals 2023 - osu!catch](2023_catch)
```
check titles here against the articles, as some don't match and some needed to be corrected in the articles

Walavouchey reviewed (changes requested):
after applying these:

- all headings should be sentence case, not title case
- bold tiebreakers
- i've done this for some spots in the reviews, but wherever you have 4K/7K splits, put them below the level 2 headings. for example:

```markdown
## Participants

### osu!mania 4K

### osu!mania 7K
```

- all of these are missing badge images

i'll check the rulesets afterwards

Walavouchey commented on wiki/Tournaments/oPHN/2019_4K/en.md:
```diff
@@ -62,122 +66,172 @@ The **osu! Philippines Nationals 2019 (osu!mania 4K)** (***PHN4K 2019***) is a o
+  - [sakuraburst - deconstructing nature (Vortex-) [destruction]](https://osu.ppy.sh/beatmapsets/619106#mania/1305926)
+- Tiebreaker
+  - **Rawtekk - Amber's Love Was Like A Marble (Billain Remix) (Evening) \[corrupted\]**
+
 ### Finals week 1
 
-**[Download the mappack here! (84.9 MB)](https://drive.google.com/file/d/1FGo-7Xy2ZhTaM_pJtdmzVOltKLF2KmlK/view?usp=sharing)**
+**[Download the mappack here! (84.9 MB)](https://drive.google.com/file/d/1FGo-7Xy2ZhTaM_pJtdmzVOltKLF2KmlK/view?usp=sharing "Google Drive")**
 
 - FreeMod
-  1. Terminal11 - Odd22 (Lenfried-) \[Choof's Masochist (LN Fix)\]
-  2. [SHIKI - Angelic layer (Vortex-) \[Distortion\]](https://osu.ppy.sh/beatmapsets/735008#mania/1551698)
-  3. [Yuyoyuppe - Palette GAMMA (Toaph Daddy) \[Colors\]](https://osu.ppy.sh/beatmapsets/947386#mania/1978712)
-  4. [Mayumi Morinaga - dreamin' feat. Ryu\* (+VOX Mix) (\_underjoy) \[UJCHAN!!\]](https://osu.ppy.sh/beatmapsets/885179#mania/1850115)
-  5. [Camellia feat. Nanahira - Ningen vs Kyuunage Robot Tama-chan no Catchball Taiketsu (Polytetral) \[Beginner\]](https://osu.ppy.sh/beatmapsets/995878#mania/2083171)
-  6. Two Steps From Hell - Star Sky (Evening) \[Ethereal\]
-  7. [YMCK - Curry Da Yo! (qqqant) \[Ricetune\]](https://osu.ppy.sh/beatmapsets/983584#mania/2058130)
-  8. [Casey Edwards feat. Ali Edwards - Devil Trigger (Valedict) \[Rebellion\]](https://osu.ppy.sh/beatmapsets/889477#mania/1859068)
-  9. [bj.HaLo - Gekitsui -HELL AIR FORCE- (stupud man) \[Inundation\]](https://osu.ppy.sh/beatmapsets/644532#mania/1365985)
-  10. [Tia - Deal with the devil (-Rem-) \[gamble\]](https://osu.ppy.sh/beatmapsets/995384#mania/2082239)
-  11. [Nekobolo - My Last (KatayokuNoTori) \[Final (o2 Style)\]](https://osu.ppy.sh/beatmapsets/622875#mania/1313335)
-  12. [loz - Cinderella Cage -Trancecore Mix- (Kamikaze) \[Tails' vs Kami's Lunatic\]](https://osu.ppy.sh/beatmapsets/925063#mania/2022076)
-  13. sakuzyo - Black Lair (RemFangirl) \[Depths\]
-  14. [ONE - Future (arcwinolivirus) \[4K SV Innovation\]](https://osu.ppy.sh/beatmapsets/708785#mania/1498464)
-  15. saradisk - 168 - 401 (Kamikaze) \[Subnautical Activities\]
+  - Terminal11 - Odd22 (Lenfried-) \[Choof's Masochist (LN Fix)\]
+  - [SHIKI - Angelic layer (Vortex-) \[Distortion\]](https://osu.ppy.sh/beatmapsets/735008#mania/1551698)
+  - [Yuyoyuppe - Palette GAMMA (Toaph Daddy) \[Colors\]](https://osu.ppy.sh/beatmapsets/947386#mania/1978712)
+  - [Mayumi Morinaga - dreamin' feat. Ryu\* (+VOX Mix) (\_underjoy) \[UJCHAN!!\]](https://osu.ppy.sh/beatmapsets/885179#mania/1850115)
+  - [Camellia feat. Nanahira - Ningen vs Kyuunage Robot Tama-chan no Catchball Taiketsu (Polytetral) \[Beginner\]](https://osu.ppy.sh/beatmapsets/995878#mania/2083171)
+  - Two Steps From Hell - Star Sky (Evening) \[Ethereal\]
+  - [YMCK - Curry Da Yo! (qqqant) \[Ricetune\]](https://osu.ppy.sh/beatmapsets/983584#mania/2058130)
+  - [Casey Edwards feat. Ali Edwards - Devil Trigger (Valedict) \[Rebellion\]](https://osu.ppy.sh/beatmapsets/889477#mania/1859068)
+  - [bj.HaLo - Gekitsui -HELL AIR FORCE- (stupud man) \[Inundation\]](https://osu.ppy.sh/beatmapsets/644532#mania/1365985)
+  - [Tia - Deal with the devil (-Rem-) \[gamble\]](https://osu.ppy.sh/beatmapsets/995384#mania/2082239)
+  - [Nekobolo - My Last (KatayokuNoTori) \[Final (o2 Style)\]](https://osu.ppy.sh/beatmapsets/622875#mania/1313335)
+  - [loz - Cinderella Cage -Trancecore Mix- (Kamikaze) \[Tails' vs Kami's Lunatic\]](https://osu.ppy.sh/beatmapsets/925063#mania/2022076)
+  - sakuzyo - Black Lair (RemFangirl) \[Depths\]
+  - [ONE - Future (arcwinolivirus) \[4K SV Innovation\]](https://osu.ppy.sh/beatmapsets/708785#mania/1498464)
+  - saradisk - 168 - 401 (Kamikaze) \[Subnautical Activities\]
```
this is incorrect, all mappools *should* be numbered lists (see every other tournament article)

this applies to all articles in this pr

topecnz replied:
done

Walavouchey commented on wiki/Tournaments/oPHN/2019_4K/en.md:
```diff
@@ -46,10 +50,10 @@ The **osu! Philippines Nationals 2019 (osu!mania 4K)** (***PHN4K 2019***) is a o
+- [YouTube](https://www.youtube.com/channel/UCkV4V1OVO6gOGxtH51glH0Q/live "YouTube")
+- [Twitch](https://www.twitch.tv/osuliveph "Twitch")
+- [Challonge](https://challonge.com/oPHN4K19 "Challonge")
+- [osu! Philippines Tournament Discord server](https://discord.gg/9ZUMmwS "Discord")
```
we don't use tooltips for links on the wiki like this. it's also redundant here

topecnz replied:
it seems already updated this part.

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+### General rules
+
+- ScoreV2 will be used for this tournament.
+- Players are allowed to join the 10 minutes before the scheduled match.
```
```suggestion
- Players are allowed to join 10 minutes before the scheduled match.
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+- ScoreV2 will be used for this tournament.
+- Players are allowed to join the 10 minutes before the scheduled match.
+- Players cannot nullify a beatmap under any circumstances
+- Players will be given 10 minutes to join the lobby, failing to show up will result to win by default(WBD).
```
```suggestion
- Players will be given 10 minutes to join the lobby, and failing to show up will result in a win by default (WBD).
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+- Players are allowed to join the 10 minutes before the scheduled match.
+- Players cannot nullify a beatmap under any circumstances
+- Players will be given 10 minutes to join the lobby, failing to show up will result to win by default(WBD).
+- The first 30 seconds of the match can be aborted by the referee if any unfortunate event happen in player's end such as sudden internet disconnections or blackout this is only allowed once in a match per player/participants and they will be given 3 minutes to get back in order to restart the round.
```
weird wording, grammatical issues

```suggestion
- A match may be aborted by a referee if any mishaps happen during the first 30 seconds of a match, such as for sudden internet disconnections or blackouts. This is only allowed once in a match per player and they will be given 3 minutes to get back in order to restart the round.
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+- Players cannot nullify a beatmap under any circumstances
+- Players will be given 10 minutes to join the lobby, failing to show up will result to win by default(WBD).
+- The first 30 seconds of the match can be aborted by the referee if any unfortunate event happen in player's end such as sudden internet disconnections or blackout this is only allowed once in a match per player/participants and they will be given 3 minutes to get back in order to restart the round.
+- Players are allowed to join the 10 minutes before the scheduled match.
+- Players cannot nullify a beatmap under any circumstances
+- Players will be given 10 minutes to join the lobby, failing to show up will result to win by default(WBD).
```
duplicate lines (what)

```suggestion
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+
+### Qualifier guidelines
+
+- In order to advance to the knockout stage, Players need to accumulate enough scores for each mod bracket in order to be able to proceed to knockout stage.
+- Participants will be split into lobbies with different time within the day.
+- Participants has the discretion to change their lobbies 2 days before the start of the qualifiers.
+- As the qualifiers on each lobby starts, the map pool will traverse in this pattern:
+  - NM 1 > HD 1 > HR 1 > DT 1 > NM 2 > HD 2 > HR 2 > DT 2
+- 2 Rotations will occur per each session.
+- If a tied overall qualifier scores occurred, the seeding of each involved player will based on their current performance points as of the day when the qualifiers started
```
```suggestion
- In order to advance to the knockout stage, players need to accumulate enough scores for each mod bracket.
- Participants will be split into lobbies for different times during the day.
- Participants have the discretion to change their lobbies 2 days before the start of the Qualifiers.
- As the Qualifiers on each lobby starts, the map pool will follow this pattern:
  - NM 1 -> HD 1 -> HR 1 -> DT 1 -> NM 2 -> HD 2 -> HR 2 -> DT 2
- 2 rotations will occur per each session.
- In case of ties in Qualifier scores, the seeding of affected players will be based on their current performance points as of the day when the Qualifiers started.
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+
+### Knockout guidelines
+
+- Knockouts will be double eliminations, this means there will be losers bracket, and each players are twice to beat.
+- Game sizes on each stages are the following:
```
```suggestion
- Knockouts will be double eliminations, which means there will be losers' bracket, and each player needs to lose twice in order to be eliminated.
- Match conditions for each stage are as follows:
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+- Knockouts will be double eliminations, this means there will be losers bracket, and each players are twice to beat.
+- Game sizes on each stages are the following:
+
+| Round Stage | Best of | First to |
```
```suggestion
| Stage | Best of | First to |
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+| Quarterfinals | 7 | 4 |
+| Semifinals | 9 | 5 |
+| Finals | 9 | 5 |
+| Grandfinals | 11 | 6 |
```
```suggestion
| Grand Finals | 11 | 6 |
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+| Finals | 9 | 5 |
+| Grandfinals | 11 | 6 |
+
+- When a Player quits the match, They will be given 5 minutes to go back to the match lobby, failing to comply will result to a disqualification.
+- Players are only allowed to ban 1 map per match.
+- Players who gets the highest roll will pick first, otherwise ban first.
```
```suggestion
- When a player quits a match, they will be given 5 minutes to go back to the match lobby. Failing to do so will result to a disqualification.
- Players are only allowed to ban 1 map per match.
- Players who get the highest roll will pick first, and their opponent will ban first.
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+- In order to advance to the knockout stage, Players need to accumulate enough scores for each mod bracket in order to be able to proceed to knockout stage.
+- Participants will be split into lobbies with different time within the day.
+- Participants has the discretion to change their lobbies 2 days before the start of the qualifiers.
+- As the qualifiers on each lobby starts, the map pool will traverse in this pattern:
```
```suggestion
- As the qualifiers on each lobby starts, the mappool will traverse in this pattern:
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+
+### Mappool format
+
+The mappool format for qualifiers that this tournament will be using will be the following:
```
```suggestion
#### Qualifiers
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+| Hard Rock (HR) | 2 |
+| Double Time (DT) | 2 |
+
+The mappool format for knockouts will use this format throughout the rounds:
```
```suggestion
#### Knockout stage
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+
+### Mod usage rules
+
+- **Players are required to use No Fail mod before starting the matches.**
+- Tie Breaker will be played with NoMod. The use of HD mod will be allowed.
+- At any map, players are not allowed to use the following mods:
```
```suggestion
- **Players are required to enable No Fail before starting matches.**
- The tiebreaker will be played with No Mod, but the use of Hidden will be allowed.
- During any map, players are not allowed to use the following mods:
```

Walavouchey commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,331 @@
+  - Flashlight
+  - Relax
+  - Auto
+- When playing either Double Time or HardRock maps, players are allowed to pick Hidden mod.
```
```suggestion
- When playing either Double Time or Hard Rock maps, players are allowed to pick the Hidden mod.
```

Walavouchey reviewed (commented):
another incremental review that i had lying around (yes this is gonna take a while, pleas help)

topecnz commented on wiki/Tournaments/oPHN/2022_taiko/en.md:
```diff
@@ -0,0 +1,335 @@
+
+### General rules
+
+1. This is an osu!taiko, double-elimination, IVI tournament, which is open for players globally ranked from #1000 to #10000 (inclusive) with the Philippine flag on their profile. Filipino players without the Philippine flag on their profile may present a proof of their Philippine citizenship in order to join. Hosts, poolers (custom mappers and playtesters included), and referees are strictly not  ...
```
`Philippine` word is a valid english word tho.

topecnz commented on wiki/Tournaments/oPHN/2022_mania/en.md:
Maybe ask surf.

Albionthegreat replied:
```suggestion
  - During the Group Stage, the losing player will be directed by the stipulations outlined in the [group stage](#group-stage) section of this ruleset.
```

topecnz commented on wiki/Tournaments/oPHN/2022_mania/en.md:
Maybe ask surf.

Albionthegreat replied:
```suggestion
A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the 10 minute grace period mentioned in the [match protocol](#match-protocol) section will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+Players will have 2 minutes to pick a map and 2 minutes to get ready. Players also allowed to pick their opponent's protect.
+
+- In case a player takes more than 2 minutes to pick a map, they will receive a warning from the referee, on subsequent occurrences the referee will pick a random map available to pick from the mappool using !roll X, where X is the number of available maps to pick in the mappool.
+- Each player is entitled to an optional (not required to be used) time out of two minutes when its their turn to pick. This time out period will only be allowed once per player per match.
```
```suggestion
- Each player is entitled to an optional (not required to be used) timeout of two minutes when its their turn to pick. This timeout period will only be allowed once per player per match.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+
+Matches will have a two disconnection limit - upon the third disconnection, a point will be handed immediately to the player still active in the multiplayer room.
+
+A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the 10 minute grace period mentioned in Section 2.1 (Protocol) will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.
```
```suggestion
A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the 10 minute grace period mentioned in [match protocol](#match-protocol) will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.
```

Albionthegreat reviewed (changes requested):
I also noticed the Prizes section in the different tournaments does not have a badge added. Perhaps these can be added to the img folder and displayed?

Albionthegreat commented on wiki/Tournaments/oPHN/2022_mania/en.md:
```diff
@@ -0,0 +1,789 @@
+| Round of 16 | 2022-06-18/2022-06-19 |
+| Quarterfinals | 2022-06-25/2022-06-26 |
+| Semifinals | 2022-07-02/2022-07-03 |
+| Finals week 1 | 2022-07-09/2022-07-10 |
+| Finals week 2 | 2022-07-16/2022-07-17 |
```
```suggestion
| Finals | 2022-07-09/2022-07-10 |
| Grand Finals | 2022-07-16/2022-07-17 |
```
The page mentions both 'Grand Finals' and 'Finals week 2'. This, along with the table in the rules section, are the only places where 'Finals week 2' is mentioned. Given that the sheet also calls it Grand Finals, I think we can go with that.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2022_mania/en.md:
```diff
@@ -0,0 +1,789 @@
+
+## Mappools (osu!mania 4K)
+
+### Grand finals
```
```suggestion
### Grand Finals
```
While headers are sentence case, the term 'Grand Finals' should have both Grand and Finals capitalised.

Albionthegreat commented on wiki/Tournaments/oPHN/2022_mania/en.md:
```suggestion
| Stage | Format | Mappool size |
| :-- | :-- | :-- |
| Group stage | Best of 7 | 10 FM, 1 TB |
| Round of 16 | Best of 9 | 12 FM, 1 TB |
| Quarterfinals | Best of 11 | 14 FM, 1 TB |
| Semifinals | Best of 13 | 16 FM, 1 TB |
| Finals | Best of 13 | 16 FM, 1 TB |
| Grand Finals | Best of 13 | 16 FM, 1 TB |
```
Same thing as PHNM 2023

Albionthegreat commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,640 @@
+
+### 4K
+
+#### Grand finals
```
```suggestion
#### Grand Finals
```
While headers are sentence case, the term 'Grand Finals' should have both Grand and Finals capitalised.

Albionthegreat commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,640 @@
+
+## Ruleset
+
+### Tournament organization
```
```suggestion
### Tournament organisation
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,640 @@
+
+All players will have two chances at each qualifier notechart, in which the highest of the two runs on each qualifier notechart will be used to calculate your final score.
+
+Players will be organized into groups based on their availability. Because qualification is not a process that involves 1 on 1 matchups, it is actually possible to have 8 players go through the qualification process at once. If this is a confusing process, do not worry: simply adhere to the schedule that will be released and follow referee instructions during the match.
```
```suggestion
Players will be organised into groups based on their availability. Because qualification is not a process that involves 1 on 1 matchups, it is actually possible to have 8 players go through the qualification process at once. If this is a confusing process, do not worry: simply adhere to the schedule that will be released and follow referee instructions during the match.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+### Registration
+
+- Only players with the Philippine flag on their profile are eligible to play. Players who don’t have the Philippine flag on their profile can present a proof of their Philippine Citizenship to the staff in order to be eligible to play.
+- Staff Members **who are NOT Referees, Mappool Selectors, Hosts, nor are part of any managing role** are permitted to join the tournament. [Read the FAQ](/wiki/Tournaments/Official_support#faq) about what is considered as “staff” for more details.
```
```suggestion
- Staff Members **who are NOT Referees, Mappool Selectors, Hosts, nor are part of any managing role** are permitted to join the tournament. [Read the FAQ](/wiki/Tournaments/Official_support#faq) about what is considered as "staff" for more details.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+### Tournament format
+
+The tournament format was 1v1, double-elimination tournament. Players play through qualifiers, and the top 32 players will proceed to the Round of 32. 
```
```suggestion
The tournament format was 1v1, double elimination. Players play through Qualifiers, and the top 32 players will proceed to the Round of 32. 
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+### Pool size
+
+Note: The star ratings listed below are the range of  star ratings per round.
```
```suggestion
Note: The star ratings listed below are the range of star ratings per round.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+| Quarterfinals | 11 | 5 | 4 | 4 | 4 | 1 | 5.2*-6.6* |
+| Semifinals | 11 | 5 | 4 | 4 | 4 | 1 | 5.5*-6.8* |
+| Finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.8*-7.1* |
+| Grand finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.9*-7.3* |
```
```suggestion
| Grand Finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.9*-7.3* |
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+Note: The star ratings listed below are the range of  star ratings per round.
+
+| Round stage | Best of | NM | HD | HR | DT | TB | Star rating |
```
```suggestion
| Round | Best of | NM | HD | HR | DT | TB | Star rating |
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+### Tournament Rules
+
+- All schedules in the tournament are default to Philippine Standard Time (+8 UTC).
```
```suggestion
- All schedules in the tournament are default to Philippine Standard Time (UTC+8).
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+| Finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.8*-7.1* |
+| Grand finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.9*-7.3* |
+
+### Tournament Rules
```
```suggestion
### Tournament rules
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+- Banning two maps from the same mod category is not allowed.
+- Any player who displays unsportsmanlike behavior will receive a warning/disqualification.
+
+### Qualifier Rules
```
```suggestion
### Qualifier rules
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+- In case of disconnection (while the map was being played), the player may replay the missed map **one** time later on.
+- The top 32 players by **average map rank** will proceed to RO32.
+
+### Match Procedure
```
```suggestion
### Match procedure
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+- Referees will notify and invite both players into the lobby before the designated schedule. 
+- There will be no warmups.
+- Players will “!roll” and the winner decides between first pick, second ban or second pick, first ban.
```
```suggestion
- Players will `!roll` and the winner decides between first pick, second ban or second pick, first ban.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+### Registration
+
+- Only players with the Philippine flag on their profile are eligible to play. Players who don’t have the Philippine flag on their profile can present a proof of their Philippine Citizenship to the staff in order to be eligible to play.
```
```suggestion
- Only players with the Philippine flag on their profile are eligible to play. Players who don't have the Philippine flag on their profile can present a proof of their Philippine Citizenship to the staff in order to be eligible to play.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+
+## Mappool
+
+### Grand finals
```
```suggestion
### Grand Finals
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+
+## Match results
+
+### Grand finals
```
```suggestion
### Grand Finals
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_osu/en.md:
```diff
@@ -0,0 +1,468 @@
+
+The tournament format was 1v1, double-elimination tournament. Players play through qualifiers, and the top 32 players will proceed to the Round of 32. 
+
+Players get 1 ban on RO32 and RO16 and 2 bans on quarterfinals onwards. All matches are played with Score v2 and NF, head-to-head mode.
```
```suggestion
Players get 1 ban on RO32 and RO16 and 2 bans on Quarterfinals onwards. All matches are played with ScoreV2 and NF in Head-to-head mode.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+
+1. This is an osu!taiko, double-elimination, 1v1 tournament open for players globally ranked from #500 to #10000 (inclusive) with the Philippine flag on their profile. Filipino players without the Philippine flag on their profile may present a proof of their Philippine citizenship in order to join
+2. Hosts, poolers, custom mappers, and playtesters are strictly not allowed to play. Also, players are allowed to staff as referee, streamer, or commentator once they are eliminated from the tournament.
+3. Matches will use ScoreV2 and NoFail in Head-to-Head mode.
```
```suggestion
3. Matches will use ScoreV2 and No Fail in Head-to-Head mode.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+2. Hosts, poolers, custom mappers, and playtesters are strictly not allowed to play. Also, players are allowed to staff as referee, streamer, or commentator once they are eliminated from the tournament.
+3. Matches will use ScoreV2 and NoFail in Head-to-Head mode.
+4. Players and staff must be treated with respect. Instructions from the hosts and referees are to be followed. Abusing the rules and poor sportsmanship may result in disqualification from the entire tournament.
+5. Any modification from the ruleset will be announced. Players must be active on the tournament's discord server in the case of these modifications.
```
```suggestion
5. Any modification from the ruleset will be announced. Players must be active on the tournament's Discord server in the case of these modifications.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+
+### Mappool structure
+
+- There will be separate mappools for each stage. Each mappool contains the following mod brackets: NoMod, Hidden, HardRock, DoubleTime, ForceMod, and Tiebreaker. There will be no Tiebreaker for Qualifiers.
+  - DoubleTime can be played as Nightcore if all players in the lobby agree to it.
+  - ForceMod will be played either with Hidden, HardRock, or Hidden + HardRock.
+  - Tiebreaker will be played with NoMod and will be only played in case of a tie.
```
```suggestion
- There will be separate mappools for each stage. Each mappool contains the following mod brackets: No Mod, Hidden, Hard Rock, Double Time, Force Mod, and Tiebreaker. There will be no Tiebreaker for Qualifiers.
  - Double Time can be played as Nightcore if all players in the lobby agree to it.
  - Force Mod will be played either with Hidden, Hard Rock, or Hidden + Hard Rock.
  - Tiebreaker will be played with No Mod and will be only played in case of a tie.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+  - Tiebreaker will be played with NoMod and will be only played in case of a tie.
+- The win condition and mappool size per stage are as follows:
+
+| Round stage | Best of | NM | HD | HR | DT | FM | TB |
```
```suggestion
| Round | Best of | NM | HD | HR | DT | FM | TB |
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+| Quarterfinals | 11 | 5 | 2 | 2 | 2 | 3 | 1 |
+| Semifinals | 11 | 5 | 2 | 2 | 2 | 3 | 1 |
+| Finals | 13 | 6 | 2 | 2 | 2 | 3 | 1 |
+| Grand finals | 13 | 6 | 2 | 2 | 2 | 3 | 1 |
```
```suggestion
| Grand Finals | 13 | 6 | 2 | 2 | 2 | 3 | 1 |
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+
+### Qualifiers regulations
+
+1. Players must sign up to one of the qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifiers mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time. A player that doesn’t pick a time slot before Thursday 23:59 will be disqualified.
```
```suggestion
1. Players must sign up to one of the Qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifiers mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time. A player that doesn’t pick a time slot before Thursday 23:59 will be disqualified.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+### Qualifiers regulations
+
+1. Players must sign up to one of the qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifiers mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time. A player that doesn’t pick a time slot before Thursday 23:59 will be disqualified.
+2. There will be no warmups for all qualifiers lobbies. The mappool will be played twice during a lobby, and will be played in the following order: NM1, NM2, NM3, HD1, HR1 ,DT1, FM1. A 5-minute break will be offered between the first and second run of the mappool, which can be skipped if all players in the lobby are ready. A player may skip certain maps during their second run if they so desire.
```
```suggestion
2. There will be no warmups in any Qualifier lobby. The mappool will be played twice during a lobby, and will be played in the following order: NM1, NM2, NM3, HD1, HR1 ,DT1, FM1. A 5-minute break will be offered between the first and second run of the mappool, which can be skipped if all players in the lobby are ready. A player may skip certain maps during their second run if they so desire.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+
+### Match regulations
+
+1. Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use DoubleTime if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.
```
```suggestion
1. Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use Double Time if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_taiko/en.md:
```diff
@@ -0,0 +1,337 @@
+### Schedule guidelines
+
+1. Schedules in this tournament are defaulted to UTC+8 (Philippine Standard Time).
+2. Each stage will be held on a single weekend. It is possible to reschedule on a Friday or Monday.
```
```suggestion
2. Each stage will be held on a single weekend. It is possible to reschedule to a Friday or Monday.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2023_mania/en.md:
```diff
@@ -0,0 +1,638 @@
+
+## Ruleset
+
+### Tournament Organization
```
Except for headers saying 'Grand Finals' I should add.

Albionthegreat commented on wiki/Tournaments/oPHN/2023_catch/en.md:
```diff
@@ -57,11 +57,11 @@ The osu! Philippines Nationals 2023 - osu!catch is ran by various community memb
 ### General rules
 
 - Score V2 will be used.
```
```suggestion
- ScoreV2 will be used.
```

Albionthegreat commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,328 @@
+- ::{ flag=AU }:: [CoolEVT](https://osu.ppy.sh/users/3252653)
+- ::{ flag=PH }:: [Laqure](https://osu.ppy.sh/users/1642435)
+- ::{ flag=PH }:: [Chloebe](https://osu.ppy.sh/users/4577865)
+- ::{ flag=UK }:: [arwab](https://osu.ppy.sh/users/16496103)
```
```suggestion
- ::{ flag=GB }:: [arwab](https://osu.ppy.sh/users/16496103)
```

Albionthegreat commented on wiki/Tournaments/oPHN/2022_catch/en.md:
```diff
@@ -0,0 +1,328 @@
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | :{ flag=PH }:: [-Rustyy](https://osu.ppy.sh/users/16355636) |
```
```suggestion
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [-Rustyy](https://osu.ppy.sh/users/16355636) |
```

TicClick reviewed (approved):
this should be ready now -- big thanks to everyone involved, authors and reviewers alike
(and please split your further PRs into smaller chunks)
